code/pizzabot.js

```diff
@@ -1,7 +1,73 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
-//Put your Javscript code here:
+const submitForm = document.getElementById("form-action");
+
+document.getElementById("welcomeMessage").innerHTML=(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+
+/*A function that checks if userinput-choice is on menu. Returns true or false. The argument 
+is the input from the form in html and is passed as a value to the parameter "chosenPizza" in the js function. 
+Also uses the toLowerCase method to allow for users using capitals and lower case. Stores the true
+value in orderName. Is checked in if-statement: if true a particular code is run. If false, another code is run. ????*/ 
+const validateOrderName = (chosenPizza) => {
+   if (chosenPizza.toLowerCase() === vegetarian.toLowerCase() || chosenPizza.toLowerCase() === pepperoni.toLowerCase() || chosenPizza.toLowerCase() === hawaiian.toLowerCase()) {
+      return true;
+   } else {
+      return false;
+   }
+};
+
+//Function that provides us a totalprice of order by calculating price and number of pizzas. Will run of if-statement is true.
+const calculateTotalCost = (pizzaPrice, orderQuantity) => {
+   orderTotal = (pizzaPrice * orderQuantity);
+   return orderTotal;
+}
+
+//Function that provdies the total cooking time depending on number of pizzas. Will run if if-statement is true.
+const calculateCookingTime = (quantity) => {
+   if (quantity <= 2) {
+      return 10; 
+   } else if (quantity > 2 && quantity <= 5) {
+      return 15;
+   } else {
+      return 20;
+   }
+};
```

Comment:
Great job :) I forgot to mention one thing - if we pass only one parameter to our function, instead o f writing it like this : 

`const calculateCookingTime = (quantity) => { ... }`

We can omit parentheses wrapping our `quantity` parameter and write :

`const calculateCookingTime = quantity => { ... }`

It's once again clean code approach. You just have to remember that we can apply this technique ONLY if we have one argument